### PR TITLE
Sonarbranch sonar corrections review

### PR DESCRIPTION
**Pull request by kovacsge11**
Review my code corrections caused by sonar

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 */
 public class TorpedoStore {
 
-  private double FAILURE_RATE = 0.0;
+  private double failureRate = 0.0;
```
Comment (dorob) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:10]: ok

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ public boolean fire(int numberOfTorpedos){
     // simulate random overheating of the launcher bay which prevents firing
     double r = generator.nextDouble();
 
-    if (r >= FAILURE_RATE) {
+    if (r >= failureRate) {
```
Comment (dorob) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:39]: ok

File: src/main/java/hu/bme/mit/spaceship/GT4500.java [hunk 3]
```diff
@@ -79,10 +83,8 @@ public boolean fireTorpedo(FiringMode firingMode) {
       case ALL:
         // try to fire both of the torpedo stores
         //implemented yay
-	if (! primaryTorpedoStore.isEmpty() && ! secondaryTorpedoStore.isEmpty()) {
-		if(primaryTorpedoStore.fire(1)){
-			firingSuccess = secondaryTorpedoStore.fire(1);		
-		}
+	if (allCondition) {
```
Comment (dorob) [inline src/main/java/hu/bme/mit/spaceship/GT4500.java:86]: good, better readability

File: src/main/java/hu/bme/mit/spaceship/GT4500.java [hunk 4]
```diff
@@ -37,19 +37,23 @@ public boolean fireLaser(FiringMode firingMode) {
...
+    secondaryTorpNotEmpty = ! secondaryTorpedoStore.isEmpty();
+    primaryTorpNotEmpty = ! primaryTorpedoStore.isEmpty();
+    allCondition = primaryTorpNotEmpty && secondaryTorpNotEmpty && primaryTorpedoStore.fire(1)
+	
     switch (firingMode) {
       case SINGLE:
         if (wasPrimaryFiredLast) {
           // try to fire the secondary first
-          if (! secondaryTorpedoStore.isEmpty()) {
+          if (secondaryTorpNotEmpty) {
```
Comment (dorob) [inline src/main/java/hu/bme/mit/spaceship/GT4500.java:49]: good, simpler than the other

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 5]
```diff
@@ -36,7 +36,7 @@ public boolean fire(int numberOfTorpedos){
     // simulate random overheating of the launcher bay which prevents firing
     double r = generator.nextDouble();
 
-    if (r >= FAILURE_RATE) {
```
Comment (dorob) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:39]: In 22 and 24 row FAILURE_RATE name still exists

**Review by dorob — CHANGES_REQUESTED** [review]
1 change need, but great

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 6]
```diff
@@ -19,9 +19,9 @@ public TorpedoStore(int numberOfTorpedos){
     String failureEnv = System.getenv("IVT_RATE");
     if (failureEnv != null){
       try {
-        FAILURE_RATE = Double.parseDouble(failureEnv);
+        failureRate = Double.parseDouble(failureEnv);
       } catch (NumberFormatException nfe) {
-        FAILURE_RATE = 0.0;
+        failureRate = 0.0;
```
Comment (dorob) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:24]: ok, good

**Comment by kovacsge11** [comment]
Corrected all bugs.

**Review by dorob — COMMENTED** [review]
everything is ok

**Pull request merged**